### PR TITLE
Update `ContactPatientBottomSheet` Behaviour

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.clubhouse.io/simpledotorg/story/3714/update-contactpatientbottomsheet-behaviour?vc_group_by=day&ct_workflow=all&cf_workflow=500000005

P.S. To test this feature out.. 
- Change `OverdueListChanges` feature flag to `true`
- Add a new patient with high bp and no phone number with next visit date as tomorrow. 
- Then add `val date = LocalDate.now(userClock).plusDays(5)` in `OverdueScreen` line 133
- Also add `LocalDate.now(clock).plusDays(5)` in `ContactPatientEffectHandler` line 70

### CODE REVIEW

File: app/src/debug/res/layout/contactpatient_callpatient.xml [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+    android:layout_marginTop="@dimen/spacing_32"
+    android:background="@color/simple_light_grey"
+    android:visibility="gone"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toBottomOf="@id/removeFromOverdueListTextView">
+
+    <TextView
+      android:id="@+id/noPhoneNumberBottomTextView"
+      style="@style/TextAppearance.Simple.Button.Big"
```
Comment (msasikanth) [inline app/src/debug/res/layout/contactpatient_callpatient.xml:315]: We are already applying text appearance for this text, so we don't need to directly reference the style here.

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -10,6 +10,7 @@
   <dimen name="spacing_10">10dp</dimen>
   <dimen name="spacing_12">12dp</dimen>
   <dimen name="spacing_16">16dp</dimen>
+  <dimen name="spacing_18">18dp</dimen>
```
Comment (msasikanth) [inline app/src/main/res/values/dimens.xml:13]: The margin-top of the divider is 16dp, so we can remove the new 18dp dimen.

**Review by msasikanth — COMMENTED** [review]
Posting some of the changes I have noticed for the newly added view. @janhavisinghh

**Comment by janhavisinghh** [comment]
@Honey14 hold off reviewing this PR making some changes.

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -12,6 +12,7 @@
   <color name="simple_green_100">#E0FFED</color>
 
   <color name="simple_dark_grey">#2F363D</color>
+  <color name="simple_dark_grey_67">#AB2F363D</color>
```
[the remark this answers is not in the document]
Comment (janhavisinghh) [reply app/src/main/res/values/colors.xml:15]: Is there a good way to find these things that I'm missing out here? 😅

File: app/src/debug/res/layout/contactpatient_callpatient.xml [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+
+  <TextView
+    android:id="@+id/patientAddressTextView"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="@dimen/spacing_24"
+    android:layout_marginTop="@dimen/spacing_16"
+    android:maxLines="2"
+    android:textAppearance="?attr/textAppearanceBody2"
+    android:textColor="#6C737A"
```
[the remark this answers is not in the document]
Comment (janhavisinghh) [reply app/src/debug/res/layout/contactpatient_callpatient.xml:35]: Shouldn't we be able to find these names from the figma file ideally? I was searching for something called grey1.

**Comment by janhavisinghh** [comment]
Divided this PR into two smaller PRs. The first one is ready to review [here](https://github.com/simpledotorg/simple-android/pull/2810). Closing this PR now will raise PR number 2 for review when the first one is merged in.

**Pull request closed without merging**